src/main/java/ita/softserve/course_evaluation_analytics/service/impl/ChatsMessagesServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package ita.softserve.course_evaluation_analytics.service.impl;
+
+import ita.softserve.course_evaluation_analytics.entity.ChatsMessages;
+import ita.softserve.course_evaluation_analytics.repository.ChatsMessagesRepository;
+import ita.softserve.course_evaluation_analytics.service.ChatsMessagesService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+@RequiredArgsConstructor
+public class ChatsMessagesServiceImpl implements ChatsMessagesService {
+    private final ChatsMessagesRepository chatsMessagesRepository;
+    @Override
+    public List<ChatsMessages> getListOfCountedMessagesByDays() {
+        return chatsMessagesRepository.getListOfMessagesByDays();
```

Comment:
I think in this case methods should have the same names in repo and service, since you just return the values from DB